rand_distr benches need `extern crate test` declared

I needed to add the line `extern crate test` in order to run the benchmarks for rand_distr, running nightly. I believe `extern crate test` is required even with 2018 edition for the case of the built-in test crate, if not please let me know what I'm doing wrong.

```console
$ rustc --version
rustc 1.42.0-nightly (0de96d37f 2019-12-19)
```

rand_distr: master branch (cloned immediately prior).

Command I ran with error:

```console
$ RUSTFLAGS='-C target-cpu=native' cargo bench
# ...
error[E0432]: unresolved import `test`
  --> rand_distr/benches/distributions.rs:15:5
   |
15 | use test::Bencher;
   |     ^^^^ use of undeclared type or module `test`
```

What I did to fix it in rand_distr/benches/distributions.rs:

```rust
#![feature(test)]

extern crate test;
```

--- a/rand_distr/benches/distributions.rs
+++ b/rand_distr/benches/distributions.rs
@@ -8,6 +8,8 @@
 
 #![feature(test)]
 
+extern crate test;
+
 const RAND_BENCH_N: u64 = 1000;
 
 use std::mem::size_of;

